gen/calc.py

```diff
@@ -725,24 +725,16 @@ def merged_dict(dict_a, dict_b):
     return yaml.dump(json.dumps(merged_checks, indent=2))
 
 
-def calculate__superuser_credentials_given(
+def validate_superuser_credentials_not_partially_given(
         superuser_service_account_uid, superuser_service_account_public_key):
     pair = (superuser_service_account_uid, superuser_service_account_public_key)
 
-    if all(pair):
-        return 'true'
-
-    if not any(pair):
-        return 'false'
-
-    # `calculate_` functions are not supposed to error out, but
-    # in this case here (multi-arg input) this check cannot
-    # currently be replaced by a `validate_` function.
```

Comment:
It appears that this this is untrue. The tests already prove it wrong.
We've come up with the conclusion that this must be wrong because we have multiple validate functions already in DC/OS that take multi-arg input + throw AssertionError on invalid input:
https://github.com/dcos/dcos/blob/5ae5d45014e2abc47a2165445de1761661cf2e62/gen/calc.py#L507